### ISSUE TITLE
No permission to view the ECV

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#952]
It seems there's a permission error on the new installation for the /static/ecv/*.ecv files, I couldn't access the NGT one (ELAN alerted me).

**Comment by susanodd** [comment]
The directory structure used to be:
src/signbank         (source code folder has a folder signbank)
env                       (virtual environment)
writable                 (writable -- the ecv folder is under this)

But now the structure is:
signbank
env
writable

There used to be an extra ".." in the paths to go up an extra level to get from src/signbank to writable
But now it only needs to go from signbank to writable (where both are siblings)

(That's just from comparing the structure to that on my own computer. So this could cause problems since the live server is different. I fixed the ones that were broken. But I don't know where the static/ecv comes from.)

**Comment by susanodd** [comment]
Okay, I noticed the permission on the writable folder itself had nothing for group other.
I made it be rx. So whatever the process is, it should be able to traverse over writable.
(There is a symbolic link to ecv under media. The static files like js and css are mapped to the media folder by Django.)

See if that works now? Thanks.

If it's still broken, @Woseseltops will need to do something. I can't make the folder "above" writable be traversable because that's actually the container itself. (This did not used to be so, as explained above, there used to be an extra level above.)
